### PR TITLE
docs: Add OGGBundle Specification

### PR DESCRIPTION
**Pull request by Nachtalb**
Can be reviewed [here](https://docs.onegovgever.ch/oggbundle-spec-draft/)

### CODE REVIEW

File: docs/public/dev-manual/oggbundle/index.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.. _kapitel-oggbundle:
+
+====================================
+Spezifikation Importformat OGGBundle
+====================================
+
+Diese DokumentE beschreibEN die Spezifikation der Datenschnittstelle zur
```
Comment (lukasgraf) [inline docs/public/dev-manual/oggbundle/index.rst]: `DokumentE beschreibEN` @Nachtalb ?

**Comment by Nachtalb** [comment]
@lukasgraf could you please upload it once more 
@phabegger I corrected the text, you can check it [here](https://docs.onegovgever.ch/oggbundle-spec-draft/dev-manual/) once it's uploaded

**Comment by lukasgraf** [comment]
@Nachtalb done

**Comment by deiferni** [comment]
whats going on here 🤔 ? can we 🚢  or is it still 🚧 ?

**Pull request merged**